docs/src/dist_lsi.rst

```diff
@@ -120,14 +120,13 @@ Distributed LSA on Wikipedia
 First, download and prepare the Wikipedia corpus as per :doc:`wiki`, then load
 the corpus iterator with::
 
-    >>> import logging, gensim, bz2
+    >>> import logging, gensim
     >>> logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', level=logging.INFO)
 
     >>> # load id->word mapping (the dictionary)
     >>> id2word = gensim.corpora.Dictionary.load_from_text('wiki_en_wordids.txt')
     >>> # load corpus iterator
     >>> mm = gensim.corpora.MmCorpus('wiki_en_tfidf.mm')
-    >>> # mm = gensim.corpora.MmCorpus(bz2.BZ2File('wiki_en_tfidf.mm.bz2')) # use this if you compressed the TFIDF output
```

Comment:
Removing this line doesn't sound right -- we still support `bz2`!

Just remove the (superfluous) `bz2.BZFile` wrapper. Dtto below.

Reply:
@piskvorky fixed in #1868